scripts/imgtool/image.py

```diff
@@ -19,6 +19,7 @@
 """
 
 from . import version as versmod
+from . import boot_record as br
```

Comment:
Since only one function is being used maybe `from .boot_record import create_sw_component_data` would be better?

Reply:
I believe it's easier, this way I can entirely abandon the module name.